internal/core/core_metadata_test.go

```diff
@@ -0,0 +1,67 @@
+package core
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMetadata(t *testing.T) {
```

Comment:
This test hasn't been updated to a new version.
It is not like we need it now it seems (in the final version Adil created a new test instead of updating the current one, but changes stayed), but it is better not to hide those changes inside refactoring.